python-sockets-tutorial/echo-server.py

```diff
@@ -10,7 +10,7 @@
     s.listen()
     conn, addr = s.accept()
     with conn:
-        print("Connected by", addr)
+        print(f"Connected by {addr}")
```

Comment:
Most other printouts start with a lower-case letter, so we could change this to be consistent.

However, I'm leaning towards suggesting switching the others to start with an upper-case letter, as that would better reflect our other practices at Real Python.